plugins/c8y_remote_access_plugin/src/config.rs

```diff
@@ -0,0 +1,43 @@
+use miette::Context;
+use miette::IntoDiagnostic;
+use std::net::IpAddr;
+
+pub struct TedgeConfig {
```

Comment:
I'd have named this the `RemoteAccessPluginConfig` to avoid any ambiguity with `tedge_config::TEdgeConfig`.